Sources/SVGPath.swift

```diff
@@ -197,16 +197,11 @@ public struct SVGPath: Hashable {
             case "-":
                 try processNumber()
                 number = "-"
-            case "a" ... "z":
+            case "a" ... "z", "A" ... "Z":
                 try processNumber()
                 try processCommand()
                 token = char
-                isRelative = true
-            case "A" ... "Z":
-                try processNumber()
-                try processCommand()
-                token = char
-                isRelative = false
+                isRelative = char >= "a" && char <= "z"
```

Comment:
Could simplify this to just `isRelative = char > "Z"` since A-Z is before a-z in the ascii table.

Reply:
Hey @nicklockwood, I've changed it with a commit, can you check it out?